Visualisation in clustering (#260)
immunomind/immunarch: vignettes/web_only/BCRpipeline.Rmd

> `.min_nuc_outside_cdr3` — filters out reads with low numbers of nucleotides from V or J gene outside of CDR3 region

Reviewer: filters out the readings with low nucleotide numbers

Reviewer 2: I noticed that we copied this description from older version of germline.R, and in new version there is more clear description of this parameter: `this parameter sets how many nucleotides should have V or J chain outside of CDR3 to be considered good for further alignment`

Reviewer 2: We can also add that "reads with too short chains are filtered out"

> `.min_nuc_outside_cdr3` — filters out reads with low numbers of nucleotides from V or J gene outside of CDR3 region

Author: ```suggestion
`.min_nuc_outside_cdr3` — this parameter sets how many nucleotides should have V or J chain outside of CDR3 to be considered good for further alignment. Reads with too short chains are filtered out
```

```{r example 5}
bcrdata$data %>%
    top(1) %>%

>     repGermline(.ref_only_first = F, .threads = 1) %>% .$full_clones %>% .$Germline.sequence

Reviewer 2: ```suggestion
    repGermline(.ref_only_first = FALSE, .threads = 1) %>% .$full_clones %>% .$Germline.sequence
```

- must contain the 'Cluster' column generated by seqCluster() function,
- must contain the 'Sequence.germline' column generated by repGermline() function.

Align sequences in a cluster can be visualized using standard functions:

```{r align visualisation}
# A name of the first cluster
align_dt$full_clones$Cluster[[1]]

# Alignment of sequences from the first cluster

> image(align_dt$full_clones$Alignment[[1]], grid = T)

Reviewer 2: ```suggestion
image(align_dt$full_clones$Alignment[[1]], grid = TRUE)
```
